### PR TITLE
Add gpu index examples:cagra and brute froce

### PR DESCRIPTION
**Pull request by cqy123456**

### CODE REVIEW

File: examples/gpu_bf.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+)
+
+dim = 128
+nb = 10000
+nq = 10
+collection_name = "GPU_BRUTE_FORCE_test"
+
+# configure milvus hostname and port
+print(f"\nCreate connection...")
+connections.connect(host="10.100.36.182", port=19530)
```
Comment (czs007) [inline examples/gpu_bf.py:19]: localhost

File: examples/milvus_gpu.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+)
+
+dim = 128
+nb = 10000
+nq = 10
+collection_name = "cagra"
+
+# configure milvus hostname and port
+print(f"\nCreate connection...")
+connections.connect(host="10.100.36.182", port=19530)
```
Comment (czs007) [inline examples/milvus_gpu.py:19]: localhost

**Review by longjiquan — APPROVED** [review]
/lgtm

**Pull request merged**